ggshield/cmd/auth/logout.py

```diff
@@ -0,0 +1,104 @@
+import click
+from requests.exceptions import ConnectionError
+
+from ggshield.core.client import create_client
+from ggshield.core.config import Config
+from ggshield.core.utils import dashboard_to_api_url
+
+
+REVOKE_FAIL_MESSAGE = (
+    "Logout failed due to an error when revoking the token, "
+    "you can skip revocation with --no-revoke to bypass"
+)
+
+
+@click.command()
+@click.option(
+    "--instance",
+    required=False,
+    type=str,
+    help="URL of the instance to logout from.",
+)
+@click.option(
+    "--no-revoke",
```

Comment:
It's better to use the `--revoke/--no-revoke` syntax here (see https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags), it's the recommended way to define boolean flags in Click. As a bonus, it avoids double negatives in the code: `no_revoke` becomes `revoke` (with a default value of False) and `if not no_revoke` becomes `if revoke`, which is more readable.